sale_financial_risk_pos_compatibility/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Sale Financial Risk in POS",
+    "version": "16.0.1.0.0",
+    "category": "Sales/Point of Sale",
+    "summary": "Sale Financial Risk control for Sales Orders created from POS",
+    "depends": ["sale_financial_risk", "pos_order_to_sale_order_delivery"],
```

Comment:
It's not really clear to me why this module depends on `pos_order_to_sale_order_delivery` instead of `pos_order_to_sale_order`

Reply:
> It's not really clear to me why this module depends on `pos_order_to_sale_order_delivery` instead of `pos_order_to_sale_order`

The financial risk module adds another widget. And this widget should be adapted to work together with the widget from pos_order_to_sale_order_delivery